src/components/PeoplePages/PeoplePages.tsx

```diff
@@ -0,0 +1,90 @@
+/* eslint-disable no-param-reassign */
+import React, { useEffect, useState } from 'react';
+import { Loader } from '../Loader';
+import { getPeople } from '../../api';
+import { Person } from '../../types/Person';
+import { PeopleTable } from '../PeopleTable';
+import { useParams } from 'react-router-dom';
+
+const setParent = (personArr: Person[]) => {
+  personArr.map(person => {
+    person.father = personArr.find(
+      personFather => person.fatherName === personFather.name,
+    );
+    person.mother = personArr.find(
+      personMother => person.motherName === personMother.name,
+    );
+  });
+};
+
+export const PeoplePages = () => {
+  const [peoples, setPeoples] = useState<Person[]>([]);
```

Comment:
```suggestion
  const [people, setPeople] = useState<Person[]>([]);
```
People are already plural